lib/styles/react-joyride.scss

```diff
@@ -172,7 +172,7 @@ $joyride-tooltip-arrow-scale: ($joyride-tooltip-arrow-size / ($joyride-tooltip-a
     &.top-right {
       margin-bottom: $joyride-tooltip-arrow-height;
 
-      .triangle {
+      .joyride-tooltip__triangle {
```

Comment:
Selector `joyride-tooltip__triangle` should be written in lowercase with hyphens